flatgeobuf/flatgeobuf.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>Specify the order of variables in a comment (and if it's take a z?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/developmentseed/cloud-optimized-geospatial-formats-guide/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>